src/drivers/char/char_dev_stub.h

```diff
@@ -20,7 +20,7 @@
 		__attribute__((unused)) static struct { \
 			void *o, *c, *i, *p; \
 		} unused##__LINE__ = { \
-			.o = open_fn, .c = close_fn, .i = idesc_op, .p = priv \
```

Comment:
Could you please extract the change to a separate commit?

Reply:
I have added a separate PR for this and removed it from here.